code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -1668,3 +1668,136 @@
 		return
 	else
 		..()
+
+//jacknoir413:Areax Third
+/obj/item/weapon/melee/baton/stunstaff
```

Comment:
Please give this a `/fluff/` in the path. So probably `/obj/items/weapons/melee/baton/fluff/stunstaff`. Find/replace should do the trick.

Reply:
I honestly have no idea if the item will even work if I do that. I guess I could just copy paste all the baton code but that kinda begs the question of why bother with having baton in the filepath. I mean if it works cool.

Reply:
It will work exactly the same.

If I have an item that's `/obj/item/thingy` and define another item named `/obj/item/thingy/this/is/a/really/long/path/but/it/exists` and change like... `name = "Crazy Item"`, it will work, look, and otherwise be exactly like `/obj/item/thingy` except for the name.

Reply:
Just to try and further explain the path inheritance for your reference, you could do...
```
/obj/item/thingy
name = "A thingy"
desc = "This is a thingy"
icon = 'the_original.dmi'

/obj/item/thingy/special
name = "A special thingy"

/obj/item/thingy/special/very_special
icon = 'glowy_thingy.dmi'
```
`/obj/item/thingy` will have what you'd expect. `/obj/item/thingy/special` will have the same desc and icon as thingy, but a new name. And `/obj/item/thingy/special/very_special` will have a special icon, AND also have the name from `/obj/item/thingy/special` and the desc from `/obj/item/thingy`.

Basically just think of it as saying "Oh, you didn't define anything special for this variable/proc/whatever on your sub-type of that path? You must want the original from the parent(s)."